packages/tx/src/types.ts

```diff
@@ -152,6 +171,36 @@ export interface TransactionInterface<T extends TransactionType> {
   errorStr(): string
 }
 
+export interface EIP2718CompatibleTxInterface<T extends TransactionType = TransactionType>
+  extends TransactionInterface<T> {
+  readonly chainId: bigint
+  readonly accessList: AccessListBytes
+  readonly AccessListJSON: AccessList
```

Comment:
Sorry to be a bit pedantic here, but access lists have really nothing to do with the generic introduction of typed transactions - and so with EIP2718 - and if we have this here, we still have something structurally wrong (e.g. using this type from here while we should have used the EIP2930Compatible type).

Generally this looks great though with these new interfaces! 🤩

Reply:
That's true, EIP2930 capabilities should be left within their own scope, will adjust.